sdk/servicebus/azure-servicebus/azure/servicebus/aio/_servicebus_receiver_async.py

```diff
@@ -688,7 +718,7 @@ async def receive_deferred_messages(
             return messages
 
     async def peek_messages(
-        self, max_message_count: int = 1, **kwargs: Any
+        self, max_message_count: int = 1, *, sequence_number: int = 0, timeout: Optional[float] = None
```

Comment:
same for async, could you help add type hint for complete/abandon/defer/dead_letter message method?